devel/query.lua

```diff
@@ -331,6 +340,16 @@ function getSelectionData()
         selection = findTable(args.table)
         path_info = args.table
         path_info_pattern = path_info
+    elseif args.json then
+        local json = require("json")
+        local json_file = string.format("%s%s", dfhack.getDFPath(), args.json)
```

Comment:
Couldn't this just use string concatenation?
```suggestion
        local json_file = dfhack.getDFPath() .. args.json
```
This would fail if `args.json` is nil, but the `elseif` has already checked that.

Actually, I might recommend this, just so users don't always need to prefix paths with `/` (it's totally fine for `//` to be in a path, so `-json /hack/...` would still work):
```suggestion
        local json_file = string.format("%s/%s", dfhack.getDFPath(), args.json)
```
or
```suggestion
        local json_file = dfhack.getDFPath() .. '/' .. args.json
```
(take your pick)

Reply:
lol I just copied it from `assign-profiles.lua`